### PR TITLE
NxFilterInput nx-text-input class - RSC-365

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-365

### CODE REVIEW

File: lib/src/components/NxFilterInput/NxFilterInput.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ export { Props } from './types';
 const NxFilterInput = forwardRef<HTMLDivElement, Props>(
     function NxFilterInput(props, ref) {
       const { value, placeholder, onChange, className, inputId, disabled, list, ...otherProps } = props,
-          classes = classnames('nx-filter-input', className, {
+          classes = classnames('nx-filter-input', 'nx-text-input', className, {
```
Comment (glenhunter) [inline lib/src/components/NxFilterInput/NxFilterInput.tsx:20]: This adds `width: 300px` which contradicts the Zeplin files which show the filters has having a width of `250px`.

**Comment by glenhunter** [comment]
`.nx-filter-input` has `display: inline-block` which is also set in `nx-text-input` so it should be removed from `nx-filter-input`.

**Comment by rpokorny** [comment]
> .nx-filter-input has display: inline-block which is also set in nx-text-input so it should be removed from nx-filter-input.

1025ce33

File: lib/src/components/NxFilterInput/NxFilterInput.tsx (hunk 1 above)
Comment (rpokorny) [reply lib/src/components/NxFilterInput/NxFilterInput.tsx:20]: > This adds width: 300px which contradicts the Zeplin files which show the filters has having a width of 250px.

@dturner32 says the 250 only applies in the table filter header.  Implemented in 7cdb2ee4

**Review by glenhunter — APPROVED** [review]

**Pull request merged**